app/src/main/res/values/strings.xml

```diff
@@ -589,4 +587,11 @@
     <string name="downloads_storage_use_saf_title">Use SAF</string>
     <string name="downloads_storage_use_saf_summary">The Storage Access Framework allows downloads to an external SD card.\nNote: some devices are not compatible</string>
     <string name="choose_instance_prompt">Choose an instance</string>
+    <string name="videos_fixed">Videos</string>
```

Comment:
Why did you rename this?

Reply:
I tried to explain in the first bullet, I'll retry
This string, which is located here in the app [screenshot](https://i.ibb.co/rtxq5yk/Screenshot-20200128-133137-New-Pipe-PR-2309-1.png), is not translated, and also in weblate, it's not available. So by renaming it, it would be automatically added in Weblate.

If you know how to trigger weblate for a specific string, it's unecessary, but I asked on IRC and TheAssasin answered he have no idea.

Reply:
I think the problem is that another plural string with the same name exists:
https://github.com/TeamNewPipe/NewPipe/blob/8e582aaeddaf8cbed36c920ecff435d24bf5bf39/app/src/main/res/values/strings.xml#L287-L290
`videos_fixed` is a bit misleading. What about `videos_string`?

Reply:
Yeah that's the reason. Also the reason why comments have the same problem. I changed to ``videos_string``.